multicluster/controllers/multicluster/leader/resourceexport_controller_test.go

```diff
@@ -87,6 +96,20 @@ func TestResourceExportReconciler_handleServiceExportDeleteEvent(t *testing.T) {
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace:         "default",
 			Name:              "cluster-a-default-nginx-service",
+			Finalizers:        []string{constants.LegacyResourceExportFinalizer},
+			Labels:            svcLabels,
+			DeletionTimestamp: &now,
+		},
+		Spec: mcsv1alpha1.ResourceExportSpec{
+			Namespace: "default",
+			Name:      "nginx",
+			Kind:      constants.ServiceKind,
+		},
+	}
+	existingResExportWithDomainQualifiedFinalizer := &mcsv1alpha1.ResourceExport{
```

Comment:
let's rename this to `existingResExport` and the other one to `existingResExportWithLegacyFinalizer`

same in other places